tests/queries/0_stateless/02417_opentelemetry_insert_on_distributed_table.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+# Tags: no-fasttest, distributed
+
+set -ue
+
+unset CLICKHOUSE_LOG_COMMENT
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CURDIR"/../shell_config.sh
+
+
+${CLICKHOUSE_CLIENT} --distributed_ddl_output_mode=none -nq "
+SYSTEM FLUSH LOGS ON CLUSTER test_cluster_two_shards;
+TRUNCATE TABLE IF EXISTS system.opentelemetry_span_log ON CLUSTER test_cluster_two_shards;
+
+DROP TABLE IF EXISTS default.dist_opentelemetry ON CLUSTER test_cluster_two_shards;
```

Comment:
And just to clarify, `clickhouse-test` will create random database for each test, and it is better not to hardcode database name in the test, since this will not allow it to run in parallel.